src/scss/components/text-input/_index.scss

```diff
@@ -8,12 +8,16 @@
 ///  +----------+---------+----------+
 ///  | [prefix] | control | [suffix] | auto
 ///  +----------+---------+----------+
+///  | help     | help    | help     | auto
+///  +----------+---------+----------+
+///  | error    | error   | error    | auto
+///  +----------+---------+----------+
```

Comment:
Did you manually test this across all five browsers?

Reply:
I have tested all five browsers(Chrome, Firefox, IE11, legacy Edge and Safari). It's working fine.